config/mainCfg_ETau_ttCR_Legacy2016.cfg

```diff
@@ -0,0 +1,82 @@
+
+variables = DNNoutSM_kl_1
+
+selections = ttCR_invMcut
+
+regions    = SR, SStight, OSinviso, SSinviso
+
+[configs]
+sampleCfg = config/sampleCfg_Legacy2016.cfg
+cutCfg    = config/selectionCfg_ETau_ttCR_Legacy2016.cfg
+# pattern   = goodsystfiles 
+
+[merge]
+# For limits
+TT        = TTfullyHad, TTfullyLep, TTsemiLep
+W         = WJets_HT_0_70, WJets_HT_70_100, WJets_HT_100_200, WJets_HT_200_400, WJets_HT_400_600, WJets_HT_600_800, WJets_HT_800_1200, WJets_HT_1200_2500, WJets_HT_2500_Inf
+EWK       = EWKWMinus2Jets_WToLNu, EWKWPlus2Jets_WToLNu, EWKZ2Jets_ZToLL
+singleT   = singleTop_top, singleTop_antitop
+TW        = TWtop, TWantitop
+ZH        = ZH_HBB_ZLL, ZH_HBB_ZQQ, ZH_HTauTau
+WH        = WplusHTauTau, WminusHTauTau
+VV        = WWTo2L2Nu, WWTo4Q, WWToLNuQQ, WZTo1L1Nu2Q, WZTo1L3Nu, WZTo2L2Q, WZTo3LNu, ZZTo2L2Nu, ZZTo2L2Q, ZZTo2Q2Nu, ZZTo4L, ZZTo4Q
+ttH       = ttHJetTononBB, ttHJetToBB
+TTX       = TTZZ, TTWW, TTWZ, TTWJetsToLNu, TTWJetsToQQ, TTZToLLNuNu
+ggH       = ggHTauTau
+VBFH      = VBFHTauTau
+VVV       = WWW, WWZ, WZZ, ZZZ
+
+data_obs  = DsingleEleB, DsingleEleC, DsingleEleD, DsingleEleE, DsingleEleF, DsingleEleG, DsingleEleH
+
+
+############################################################################################
+############################################################################################
+## parameters for QCD evaluation
+## doFitIf : condition to be respected to make rlx-to-tight QCD fit. It is used as eval(doFitIf). Use names sel, var
+
+[pp_QCD]
+#for normal QCD
+#QCDname      = QCD
+#SR           = SR
+#yieldSB      = SStight
+#shapeSB      = SSrlx
+#SBtoSRfactor = 1
+#regionD      = SSinviso
+#regionC      = OSinviso
+#doFitIf      = False
+#fitFunc      = [0] + [1]*x
+
+
+#for inverted QCD
+QCDname      = QCD
+SR           = SR
+yieldSB      = OSinviso
+shapeSB      = OSinviso
+SBtoSRfactor = 1
+doFitIf      = False
+fitFunc      = [0] + [1]*x
+regionC      = SStight
+regionD      = SSinviso
+
+############################################################################################
+############################################################################################
+## rebinning part
+## r0 = variable, selection, ...new_bins_edges...
+[pp_rebin]
+r0 = DNNoutSM_kl_1, ttCR_invMcut, 0, 1
```

Comment:
is this needed? 
same comment for all other configs

Reply:
after discussion we decided to use 20 bins in the `selection` configs and rebin to 1 single bin here